app/views/layouts/_flash_messages.html.erb

```diff
@@ -1,7 +1,10 @@
 <% flash.each do |level, log| %>
     <div class='alert alert-<%= level %>'>
-      <% if log.respond_to?(:each) %>
-        <% if log[:heading].nil? %>
+      <% if log.is_a? String %>
```

Comment:
what does this change mean? why is this changed in the "change search" pr?

Reply:
It was just a quick fix to be able to continue implementing. But after the merge with the master, I replaced the code with the new code from there.